### PR TITLE
INS-3561 Add additional info to Export model

### PR DESCRIPTION
**Pull request by Zemidar**

### CODE REVIEW

**Review by RockMother — APPROVED** [review]

File: sell/sell-api.yaml [hunk 1]
```diff
@@ -4969,6 +4969,13 @@ components:
       properties:
         id:
           $ref: "#/components/schemas/Id"
+        type:
+          type: string
+          enum:
+            - COMPANIES
+            - CONTACTS
+            - DEALS
+          description: The type of export.
```
Comment (olyavertwist) [inline sell/sell-api.yaml:4978]: description: The type of exported objects.

File: sell/sell-api.yaml [hunk 2]
```diff
@@ -4983,6 +4990,10 @@ components:
           type: integer
           format: int32
           description: The count of export file downloads.
+        createdAt:
+          type: integer
+          format: int64
+          description: The date the file was uploaded.
```
Comment (olyavertwist) [inline sell/sell-api.yaml:4996]: description: The date the file was downloaded.

File: sell/sell-api.yaml [hunk 3]
```diff
@@ -4983,6 +4990,10 @@ components:
           type: integer
           format: int32
           description: The count of export file downloads.
```
Comment (olyavertwist) [inline sell/sell-api.yaml:4992]: description: The number of times the export file was downloaded.

**Review by olyavertwist — CHANGES_REQUESTED** [review]

File: sell/sell-api.yaml (hunk 2 above)
Comment (RockMother) [reply sell/sell-api.yaml:4996]: Probably should be : The date when data was exported.

**Review by olyavertwist — APPROVED** [review]

**Pull request merged**